content/02.introduction.md

```diff
@@ -1,11 +1,11 @@
 ## Introduction
 
-Knowledge graphs are a practical resource for many real-world applications.
+Graphs are practical resources for many real-world applications.
 They have been used in social network mining to classify nodes [@doi:10.1007/978-1-4419-8462-3_5] and create recommendation systems [@doi:10.1145/3184558.3186904].
-These graphs have also been used in natural language processing to interpret simple questions and use relational information to provide answers [@arxiv:1404.4326; @doi:10.1145/3038912.3052675].
-In a biomedical setting, these graphs have been used to prioritize genes relevant to disease [@pmid:21121028; @doi:10.1101/651000; @doi:10.1109/CIBCB.2019.8791472; @doi:10.1093/bioinformatics/bty559], perform drug repurposing [@doi:10.7554/eLife.26726] and identify drug-target interactions [@doi:10.1371/journal.pcbi.1002574].
+They have also been used in natural language processing to interpret simple questions and use relational information to provide answers [@arxiv:1404.4326; @doi:10.1145/3038912.3052675].
+In a biomedical setting, graphs have been used to prioritize genes relevant to disease [@pmid:21121028; @doi:10.1101/651000; @doi:10.1109/CIBCB.2019.8791472; @doi:10.1093/bioinformatics/bty559], perform drug repurposing [@doi:10.7554/eLife.26726] and identify drug-target interactions [@doi:10.1371/journal.pcbi.1002574].
 
-Despite their utility, precisely defining a knowledge graph is a difficult task because there are multiple conflicting definitions [@raw:KG_DEF].
+Within this biomedical context, graphs can be considered knowledge graphs; althoughn, precisely defining a knowledge graph is difficult because there are multiple conflicting definitions [@raw:KG_DEF].
```

Comment:
Not letting me suggest here on phone but typo and sentence is confusing. I think you mean some are knowledge graphs (unless you mean every biomedical graph is, but that's different than below).